tools/aequatus/datatypes/aequatus_datatypes/.shed.yml

```diff
@@ -0,0 +1,9 @@
+name: aequatus_datatypes
+owner: anilthanki
+homepage_url: https://github.com/TGAC/tgac-galaxytools/tree/master/tools/aequatus/datatypes/blast_datatypes
+remote_repository_url: https://github.com/anilthanki/tgac-galaxytools/tree/master/tools/aequatus/datatypes/blast_datatypes
+description: Datatypes for Aequatus visualisation plugin 
+  This is a repository for Aequatus datatypes.
+categories:
+- visualisation
+type: unrestricted
```

Comment:
The plan is to use this data type for Visualisation at some point, we are working on visualisation plugin